src/Compilers/CSharp/Portable/Symbols/EventSymbol.cs

```diff
@@ -117,7 +117,12 @@ public EventSymbol OverriddenEvent
             {
                 if (this.IsOverride)
                 {
-                    return (EventSymbol)OverriddenOrHiddenMembers.GetOverriddenMember();
+                    if (IsDefinition)
```

Comment:
It seems reasonable that the behavior would be different for substituted symbols, but I'm concerned about the fact that ```OverriddenEvent``` and ```OverriddenOrHiddenMembers``` are out of sync.  For instance, is there a corresponding issue for hidden members?